README.md

```diff
@@ -1,45 +1,31 @@
 # PodCodar App
 
-Este projeto tem por objetivo facilitar o processo de recepção de novos membros, por estarmos enfrentando uma grande dificuldade em recepcionar com qualidade a todos que estavam chegando, resolvemos então adicionar algumas funcionalidades que podem nos ajudar.
+### First of all, welcome! :tada:
 
-## Instalação
+<br/>
 
-1. Clone o repositório
+## Objective
 
-- para fazer isso basta abrir o terminal e rodar o seguinte comando `git clone https://github.com/podcodar/app.git`
+The primary goal of this project is to streamline the onboarding process for new members. In response to the challenges posed by the increasing number of newcomers, we've taken steps to implement a range of features designed to greatly enhance our capacity to warmly and efficiently welcome them. Let's go! :rocket::star2:
 
-2. Execute o comando `yarn` para instalar as dependências do projeto.
+## Technologies we use
 
-- lembre-se de navegar até a pasta onde o projeto se encontra e então quando la estiver, execute o comando acima! Ps: para navegar entre pastas pelo terminal basta usar o comando`cd nome da pasta` para entrar na pasta e `cd ..` para regredir.
+Next.js \
+TypeScript \
+Tailwind CSS \
+Prisma \
+Jest \
+Supabase \
```

Comment:
I belive the `Supabase` line comes from previous versions of the README - back when we were tinkering on the DB of choice. We currently we use [Vercel Postgress](https://vercel.com/storage/postgres ) as our DB.

Reply:
Also, if you really want to be specific, we're using [tailwind-styled-components](https://www.npmjs.com/package/tailwind-styled-components) for styling too.

Reply:
Fine, Nicolas! I didn't know, thanks for talking. I'll complement it there. 🚀